src/framework/components/camera/component.js

```diff
@@ -388,6 +391,45 @@ class CameraComponent extends Component {
         return this._camera.projectionMatrix;
     }
 
+    /**
+     * Set camera aperture in f-stops, the default value is 1.0. Higher value means less exposure.
+     *
+     * @type {number}
+     */
+    set aperture(newValue) {
+        this._camera.aperture = newValue;
+    }
+
+    get aperture() {
+        return this._camera.aperture;
+    }
+
+    /**
+     * Set camera sensitivity in ISO, the default value is 100. Higher value means more exposure.
+     *
+     * @type {number}
+     */
+    set sensitivity(newValue) {
+        this._camera.sensitivity = newValue;
+    }
+
+    get sensitivity() {
+        return this._camera.sensitivity;
+    }
+
+    /**
+     * Set camera shutter speed in seconds, the default value is 1s. Longer shutter means more exposure.
```

Comment:
I'm really not sure about 1s as a default here. What are we going to do in the future when we perhaps add a motion blur? I think the default value should be something that would work reasonably well for future purposes.

Reply:
Addressing that here:

https://github.com/playcanvas/engine/pull/4617